src/main/java/de/themorpheus/edu/taskservice/endpoint/TaskEndpoint.java

```diff
@@ -36,8 +35,8 @@ public Object createTask(@RequestBody @Valid CreateTaskRequestDTO dto) {
 	}
 
 	@GetMapping("/lecture/{lectureNameKey}/task")
-	public Object getAllTasksFromLecture(@PathVariable @NotNull @NotEmpty @NotBlank String lectureNameKey) {
-		return this.taskController.getTasksFromLecture(lectureNameKey).getHttpResponse();
+	public Object getAllTasksFromLecture(@PathVariable @NotBlank String lectureNameKey, @RequestParam boolean showBanned) {
```

Comment:
[Make RequestParam not required](https://www.baeldung.com/spring-request-param#making-an-optional-request-parameter)